recipes/wagyu/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "wagyu" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0f0e4b78913e64083ca865f07e6aee30a24988890fc383f3003456287ed45d7f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - pybind11
+    - {{ compiler('cxx') }}
+    - conda-verify
+  host:
+    - python
+    - pip
+    - pybind11
+  run:
+    - python
```

Comment:
Upstream it looks like `reprit >=0.3.0` is needed at runtime

Reply:
I've added this package recipe to the PR as well... not sure if it needs to be a separate one?

Reply:
You can have multiple recipes in the same PR, but note the caveat in the final paragraph of https://conda-forge.org/docs/orga/guidelines.html#transferring-to-conda-forge. Basically `staged-recipes` will resolve their dependency and needed build order correctly, but there could be a race condition when they are transferred to the feedstocks where `reprit` may not be available on the CDN when `wagyu` is attempting to build and will fail requiring a manual restart of the CI to get it to work.